### ISSUE TITLE
Stop sidebar from hating me personally

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#995]
The sidebar basically swaps in and out on its own whim. The only logic I can see is that I must have harmed it in a past life, so it is wreaking its kammic vengeance on me. :face_with_head_bandage: 

We ned a more sane behavior for this. As the most minimal change:

**Increase the sidebar break point so it is the same as the MD site**

https://material.io/design/

**Comment by Aminah-SC** [comment]
:rofl:

**Comment by SamaneriVimala** [comment]
I agree, it must be kammic. It does not happen for me so I must have treated it with dignity in my previous lives.
I did make a small change some time ago that on mobile phones as soon as you click a link in the sidebar, the sidebar retracts (this was bug #856 ) . Is that what you are talking about?
Please some info as to what exactly happens and on what system.
